src/lint.nim

```diff
@@ -106,5 +117,6 @@ proc main*(fileInput, fileOutput: string) =
       discard
 
 when isMainModule:
-  var x = 13
-  main("example.nim", "out.nim")
+  import cligen
+  # var x = 13 # TODO: why?
```

Comment:
It could be removed safely. I just begin to write this library for a very short time.